### ISSUE TITLE
Local folder deleted directly upon creation

### ISSUE THREAD

**Issue by nicolas-raoul** [aegif/CmisSync#511]
- Create folder, wait
- Delete it, wait
- Recreate it
- It is deleted immediately

```
2015-01-09 16:37:38,488 [15] INFO  CmisSync.Lib.RepoBase [(null)] - Full Sync Complete: C:\Users\win7pro32bit\CmisSync\192.168.0.67\Main Repository
2015-01-09 16:37:43,316 [14] DEBUG CmisSync.Lib.Watcher [(null)] - Created: C:\Users\win7pro32bit\CmisSync\192.168.0.67\Main Repository\documentLibrary\New folder
2015-01-09 16:37:43,502 [15] INFO  CmisSync.Lib.RepoBase [(null)] - Full Sync Started: C:\Users\win7pro32bit\CmisSync\192.168.0.67\Main Repository
2015-01-09 16:37:43,523 [15] DEBUG CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Invoke a remote crawl sync
2015-01-09 16:37:43,523 [15] DEBUG CmisSync.Lib.Watcher [(null)] - Removing -> Created: documentLibrary\New folder
2015-01-09 16:37:43,690 [15] DEBUG CmisSync.StatusIconController [(null)] - StatusIconController OnSyncing
2015-01-09 16:37:43,696 [15] INFO  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Removing remotely deleted folder: C:\Users\win7pro32bit\CmisSync\192.168.0.67\Main Repository\documentLibrary\New folder
2015-01-09 16:37:43,955 [3] DEBUG CmisSync.StatusIconController [(null)] - StatusIconController Animation Elapsed 4
2015-01-09 16:37:44,038 [15] INFO  CmisSync.Lib.RepoBase [(null)] - Full Sync Complete: C:\Users\win7pro32bit\CmisSync\192.168.0.67\Main Repository
```

**Comment by nicolas-raoul** [comment]
![stacktrace](https://cloud.githubusercontent.com/assets/99590/5676908/7f81b982-9821-11e4-952d-f5653db7a1d9.png)

**Comment by nicolas-raoul** [comment]
Looks like `documentLibrary/New folder` had not been removed from database.

**Comment by nicolas-raoul** [comment]
Fixed in 06af698
